src/vm/errors/vm_errors.rs

```diff
@@ -143,4 +143,6 @@ pub enum VirtualMachineError {
     NegBuiltinBase,
     #[error("Security Error: Invalid Memory Value: temporary address not relocated: {0}")]
     InvalidMemoryValueTemporaryAddress(Relocatable),
+    #[error("{0}")]
```

Comment:
You mean to use `    #[error(transparent)]`, see: https://docs.rs/thiserror/latest/thiserror/.
Also you can use `#[from]`, also in the doc. To make type conversion easier.

```rust
#[transparent]
Other(#[from] anyhow::Error),
```